content/rancher-prime/azure/contents.lr

```diff
@@ -0,0 +1,282 @@
+   characters long".
+
+   <img src="./basiccluster.png" alt="Existing Resource Group" style="border: 2px solid  gray;">
+
+   ---
+   **NOTE**
+
+   The **Create new** resource group feature is not supported.
+
+   ---
+
+   <img src="./createnewresourcegroup_ns.png" alt="Create New Resource Group Not Supported" style="border: 2px solid  gray;">
+
+click 'Next'
+
+
+
+### Rancher Configuration
+
+1. Enter the hostname for Rancher, and it must be a fully qualified domain
+   name (FQDN). The Rancher server URL will be created using this hostname.
+
+   ---
+   **NOTE**
+
+   The IP address of the Rancher hostname must be resolvable by a public DNS.
+
+   ---
+
+2. Using the slide bar, select the number of Rancher replicas.
+3. Choose bootstrap password as it suggested by the tip. The bootstrap
+   password will be used to authenticate to Rancher dashboard during first
+   login. You should change the password after the initial login.
+
+   <img src="./rancherconfiguration.png" alt="Rancher Configuration" style="border: 2px solid  gray;">
+
+click 'Next'
+
+### Review + create
+
+
+  It will summarize the offer and link to "view automation template" (Azure Resource Manager Template) 
+
+
+     Price
+     Basics
+     Rancher Configuration
+
+click 'Create'
+
+### Deployment Complete 
+
+Deployment will be in progress. After it is completed, the Rancher Prime
+Kubernetes service extension is successfully installed.
+
+ ---
+ ** NOTE: **
+
+ In the "Extensions + applications" page, the "Provisioning State" may show
+ "Succeeded" even though the provision may still be in progress. You may monitor
+ the actual progragress by loggin into the AKS cluster and follow the
+ "rancher-cloud" deployment.
+
+ ---
+
+## Log into the Rancher dashboard
+
+You may now login to Rancher dashboard by point your browser to Rancher server
+URL **https://<Rancher hostname\>**, where **Rancher hostname** is the hostname
+you have chosen previously.
+
+ ---
+ **NOTE**
+
+ The Rancher hostname must be resolvable by public DNS. Please refer to the
+ [Prerequisites](#prerequisites) section for more details.
+
+ ---
+
+## How To Use Rancher
+
+Please refer to the [Rancher documentation](https://ranchermanager.docs.rancher.com/)
+on how to use Rancher.
+
+## Rancher Usage Record and Billing 
+
+This command will print out CSP Adapter Usage Record:
+
+```
+kubectl get CspAdapterUsageRecord rancher-usage-record -o yaml
+```
+
+Output will looks like:
+
+```
+apiVersion: susecloud.net/v1
+base_product: cpe:/o:suse:rancher:v2.7.8
+kind: CSPAdapterUsageRecord
+managed_node_count: 0
+metadata:
+  creationTimestamp: "2023-10-03T16:45:37Z"
+  generation: 480
+  name: rancher-usage-record
+  resourceVersion: "430013"
+  uid: cf47eb21-54ae-4a35-8e75-a2362a9baa4f
+reporting_time: "2023-10-04T00:44:37Z"
+```
+
+Billing will be available in Azure Portal billing
+
+Home > Cost Management <subscription> | Cost analysis
+
+
+## Upgrading a Rancher Prime PAYG Cluster
```

Comment:
can they also upgrade from the GUI, or is it CLI only?

Reply:
also, can customers opt-in to automatic upgrade?

Reply:
We can have the CLI portion in there for now. But we will update it accordingly once we have some clarity on the upgrade front.

Reply:
I am not sure if customer have the option to turn off upgrade. But that's something we can document once we have some clarity on the upgrade front.

Reply:
It does appear customer have the option to enable/disable auto upgrade. https://learn.microsoft.com/en-us/azure/virtual-machines/automatic-extension-upgrade